src/main/java/ca/wolfram/beta/symath/base/VariableNode.java

```diff
@@ -29,8 +35,8 @@ public boolean isConstant() {
     }
 
     @Override
-    public double eval(Expression.VMap variableMap) {
-        return variableMap.get(variable, 0);
+    public double eval(VMap map) {
+        return map.get(variable, 0);
```

Comment:
Personally I think we should throw an exception here. We set it to 0 before we created VMap which can throw errors